core/v2/types.go

```diff
@@ -0,0 +1,96 @@
+package corev2
+
+import (
+	"math"
+
+	"github.com/Layr-Labs/eigenda/encoding"
+)
+
+var (
+	ParametersMap = map[uint8]BlobVersionParameters{
+		0: {CodingRate: 8, ReconstructionThreshold: 0.22, NumChunks: 8192},
```

Comment:
How is `ReconstructionThreshold` determined? Does it not depend on onchain parameters?